packages/databases-collections/src/components/collections.tsx

```diff
@@ -54,11 +54,12 @@ const Collections: React.FunctionComponent<CollectionsListProps> = ({
   const isEditable = useMemo(() => {
     return isCompassInWritableMode && isInstanceWritable;
   }, [isCompassInWritableMode, isInstanceWritable]);
-  const { id: connectionId } = useConnectionInfo();
+  const connectionInfo = useConnectionInfo();
+  const { id: connectionId } = connectionInfo;
   const { openCollectionWorkspace } = useOpenWorkspace();
 
   useTrackOnChange((track: TrackFunction) => {
-    track('Screen', { name: 'collections' });
+    track('Screen', { name: 'collections' }, connectionInfo);
```

Comment:
if we need to account for connectionInfo changing (or maybe we don't) then this probably should be part of dependency?